java/007_Reverse_Integer.java

```diff
@@ -1,13 +1,13 @@
 class Solution {
     public int reverse(int x) {
-        if (x == 0) return 0;
-        long res = 0;
+        int rev = 0;
         while (x != 0) {
-            res = res * 10 + x % 10;
-            if (res > Integer.MAX_VALUE || res < Integer.MIN_VALUE)
-                return 0;
+            int pop = x % 10;
```

Comment:
Also, you can comment original solution (which is very concise), and add this new solution.